themes/aframe/source/js/docs.js

```diff
@@ -1,15 +1,13 @@
 /* global $ */
-
 (function () {
 
-var anchorHeadingsSelector = 'h2[id], h3[id], h4[id], h5[id], h6[id]';
-
 var content = $('.content');
 if (content) {
+  var anchorHeadingsSelector = 'h2[id], h3[id], h4[id], h5[id], h6[id]';
   content.addEventListener('click', function (e) {
     var el = e.target;
     if (el.matches && el.matches(anchorHeadingsSelector)) {
-      window.location.hash = '#' + el.id;
+      window.location.hash = '#' + el.id.toLowerCase();
```

Comment:
if that's the case you're covering for, then should listen for `hashchange` on `window`, not `click` on the headings